moveit_experimental/moveit_jog_arm/src/jog_calcs.cpp

```diff
@@ -268,9 +249,14 @@ void JogCalcs::stopMainLoop()
   stop_jog_loop_requested_ = true;
 }
 
-void JogCalcs::haltOutgoingJogCmds()
+void JogCalcs::pauseOutgoingJogCmds()
+{
+  pause_outgoing_jog_cmds_ = true;
+}
```

Comment:
nit, combine methods:
```suggestion
void JogCalcs::pauseOutgoingJogCmds(bool pause)
{
  pause_outgoing_jog_cmds_ = pause;
}
```

Reply:
Makes sense, I removed these (using shared variables now) and combined the pause/unpause functions in a single function `setPaused()`